src/js/generate-entries.sh

```diff
@@ -43,9 +43,4 @@ cd "$root"/tools/ts-generator/types &&
 cd "$root"/tools/ts-generator/rpc &&
   $py rpc_gen_js.py \
     --server-define-file "$root"/src/v/coproc/gen.json \
-    --output-file "$root"/src/js/modules/rpc/serverAndClients/server.ts
```

Comment:
I'd rename this commit to "Remove references to types defined in the recently deleted `gen-engine.json` file

Reply:
Done